src/main/scala/chiffre/passes/FaultInstrumentation.scala

```diff
@@ -159,19 +153,25 @@ class FaultInstrumentation(compMap: Map[String, Seq[(ComponentName, String, Clas
                 ComponentName(s"$defi.io.scan.$s",
                               ModuleName(m.name, CircuitName(c.main))) )
 
+            val wt = classOf[firrtl.passes.wiring.WiringTransform]
+            val st = classOf[chiffre.passes.ScanChainTransform]
```

Comment:
This was in the original code, and I didn't remove it because that seemed out of scope of this PR.  It's possible you removed it and I failed to notice that and added it back in when I rebased this PR on your changes.  Since it looks like it's not there anymore, I'll remove it.